services/lemmy/lemmy.service.js

```diff
@@ -0,0 +1,90 @@
+import Joi from 'joi'
+import { metric } from '../text-formatters.js'
+import { BaseJsonService, InvalidParameter } from '../index.js'
+
+const queryParamSchema = Joi.object({
+  server_fqdn: Joi.string().hostname(),
+}).required()
+
+const lemmyCommunitySchema = Joi.object({
+  community_view: Joi.object({
+    counts: Joi.object({
+      subscribers: Joi.number().required(),
+      posts: Joi.number().required(),
+      comments: Joi.number().required(),
+    }).required(),
+  }).required(),
+}).required()
+
+export default class Lemmy extends BaseJsonService {
+  static category = 'social'
+
+  static route = {
+    base: 'lemmy',
+    pattern: ':community',
+    queryParamSchema,
+  }
+
+  static examples = [
+    {
+      title: 'Lemmy',
+      namedParams: { community: 'asklemmy@lemmy.ml' },
+      staticPreview: this.render({
+        community: 'asklemmy@lemmy.ml',
+        members: 42,
+      }),
+    },
+    {
+      title: 'Lemmy',
+      namedParams: { community: 'asklemmy@lemmy.ml' },
+      queryParams: { server_fqdn: 'lemmy.world' },
+      staticPreview: this.render({
+        community: 'asklemmy@lemmy.ml',
+        members: 42,
+      }),
+    },
+  ]
+
+  static _cacheLength = 30
```

Comment:
Why cache so short? We'd normally only drop this low for stuff that is really time-sensitive. I think we can delete this and go with the default for social badges (2 mins).